### ISSUE TITLE
pytorch version conflict 

### ISSUE THREAD

**Issue by dangerous-xu** [traveller59/spconv#377]
Hi @FindDefinition,  as you said, 'since spconv 2.x doesn't depend on pytorch binary (never in future)', but I meet a pytorch version conflict with spconv

my environment:
cuda 10.2
pytorch 1.5.0
torchvision 0.6.0
spconv-cu102-2.1.4

error message as blow:
```
  File "/boostai/models/__init__.py", line 17, in <module>
    from .detection_3d.backbones.voxel_backbone import VoxelBackBone8x
  File "/boostai/models/detection_3d/backbones/voxel_backbone.py", line 5, in <module>
    import spconv.pytorch as spconv
  File "/opt/conda/lib/python3.7/site-packages/spconv/pytorch/__init__.py", line 8, in <module>
    from spconv.pytorch.conv import (SparseConv2d, SparseConv3d, SparseConvTranspose2d,
  File "/opt/conda/lib/python3.7/site-packages/spconv/pytorch/conv.py", line 27, in <module>
    import spconv.pytorch.functional as Fsp
  File "/opt/conda/lib/python3.7/site-packages/spconv/pytorch/functional.py", line 27, in <module>
    class SparseConvFunction(Function):
  File "/opt/conda/lib/python3.7/site-packages/spconv/pytorch/functional.py", line 29, in SparseConvFunction
    @amp.custom_fwd(cast_inputs=torch.float16)
AttributeError: module 'torch.cuda.amp' has no attribute 'custom_fwd'
```

it seems like pytorch version  need >= 1.6.0

**Comment by FindDefinition** [comment]
see [this](https://github.com/traveller59/spconv/blob/master/docs/FAQ.md#what-does-no-dependency-on-pytorch-mean).
I have changed code to support pytorch 1.5 but I don't have environment to test it. you can change content in "/opt/conda/lib/python3.7/site-packages/spconv/pytorch/functional.py" to https://github.com/traveller59/spconv/blob/master/spconv/pytorch/functional.py. If your problem is resolved, I will publish v2.1.8 prebuilts for this feature.

**Comment by dangerous-xu** [comment]
@FindDefinition thanks for your quick reply.
I have tested the change as you mentioned in pytorch==1.5.0 environment, and there is **NO ERROR**, thanks again
